docs/self-managed/identity/what-is-identity.md

```diff
@@ -0,0 +1,20 @@
+---
+id: what-is-identity
+title: "What is Identity?"
+sidebar_label: "What is Identity?"
+---
+
+:::note Looking for IAM documentation?
+From version 8.0.0+ the IAM component has been replaced with the Identity component.
+:::
+
+Identity is the component within the Camunda Cloud stack responsible for authentication and authorization. It allows you to manage:
```

Comment:
You can also add Roles to Users, not sure if it can be called `manage` tho

Reply:
Yea its a tricky one, in this instance I'm relying on the fact that you could consider the "managing a role" set of tasks contains a "assigning roles" task